docs/production/storage-aws.md

```diff
@@ -89,3 +89,34 @@ Several things to note here:
 -  If you want to add AWS tags to the created DynamoDB tables you
    can do it by adding a `tags` map to your schema definition. See
    [`schema configuration`](../configuration/schema-config-reference.md)
+
+## Running DynamoD in local
```

Comment:
This file covers a Cortex production setup on AWS. In a production setup we shouldn't mention the local DynamoDB application. I'm not 100% sure where this documentation should go: ideally we should provide an easy way to startup a local environment instead of documenting it. For example, if you look at the `development/` you can find a couple of docker-compose config to run a local env (running the blocks storage) with a single command. Maybe we can do something similar for the chunks storage?

Reply:
@pracucci  Docker compose sounds good, maybe I can try setting up all the three services(prometheus, cortex and dynamoDb) with docker compose). Also, I think the errors that I get are mostly related to validation check which I am covering in this PR. https://github.com/cortexproject/cortex/pull/2732

Reply:
Hi @jaybatra26. Have you had a chance to look at it?